Source/Scene/Cesium3DTile.js

```diff
@@ -851,9 +851,11 @@ define([
      * @private
      */
     Cesium3DTile.prototype.update = function(tileset, frameState) {
+        var initCommandLength = frameState.commandList.length;
         applyDebugSettings(this, tileset, frameState);
         this._content.update(tileset, frameState);
         this._transformDirty = false;
+        this._content.commandsLength = frameState.commandList.length - initCommandLength;
```

Comment:
It will be simpler if `Cesium3DTile` holds onto `this._commandsLength` instead of the contents.